File: BGTalkApp/AppLanguage.swift
import Foundation

enum AppLanguage: String, CaseIterable, Identifiable {
    case bulgarian = "bg"
    case english = "en"
    case spanish = "es"

    var id: String { rawValue }

    var displayName: String {
        switch self {
        case .bulgarian: return "🇧🇬 Bulgarian"
        case .english: return "🇺🇸 English"
        case .spanish: return "🇪🇸 Spanish"
        }
    }
}

File: BGTalkAndroid/app/src/main/java/com/kiddbg/bgtalk/AppLanguage.kt
package com.kiddbg.bgtalk

enum class AppLanguage(
    val code: String,
    val displayName: String,
    val localeTag: String
) {
    BULGARIAN("bg", "Bulgarian", "bg-BG"),
    ENGLISH("en", "English", "en-US"),
    SPANISH("es", "Spanish", "es-ES")
}
